packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -135,10 +135,9 @@ contract Multisig is PluginUUPSUpgradeable, ProposalUpgradeable, Addresslist {
     /// @return bool Returns `true` if the interface is supported.
     function supportsInterface(
         bytes4 _interfaceId
-    ) public view virtual override(PluginUUPSUpgradeable, ProposalBase) returns (bool) {
+    ) public view virtual override(PluginUUPSUpgradeable) returns (bool) {
```

Comment:
Would this still require specifying `PluginUUPSUpgradeable` ? I don't think so.

Reply:
you are right